docs/bpftool-cgroup.rst

```diff
@@ -52,98 +52,99 @@ DESCRIPTION
+        - **device** device access (since 4.15);
 
-        **bind6** call to bind(2) for an inet6 socket (since 4.17);
+        - **bind4** call to bind(2) for an inet4 socket (since 4.17);
 
-        **post_bind4** return from bind(2) for an inet4 socket (since 4.17);
+        - **bind6** call to bind(2) for an inet6 socket (since 4.17);
 
-        **post_bind6** return from bind(2) for an inet6 socket (since 4.17);
+        - **post_bind4** return from bind(2) for an inet4 socket (since 4.17);
 
-        **connect4** call to connect(2) for an inet4 socket (since 4.17);
+        - **post_bind6** return from bind(2) for an inet6 socket (since 4.17);
 
-        **connect6** call to connect(2) for an inet6 socket (since 4.17);
+        - **connect4** call to connect(2) for an inet4 socket (since 4.17);
 
-        **sendmsg4** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
+        - **connect6** call to connect(2) for an inet6 socket (since 4.17);
+
+        - **sendmsg4** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
         unconnected udp4 socket (since 4.18);
 
-        **sendmsg6** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
+        - **sendmsg6** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
         unconnected udp6 socket (since 4.18);
 
-        **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
+        - **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
         an unconnected udp4 socket (since 5.2);
-        
-        **recvmsg6** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
+
+        - **recvmsg6** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
         an unconnected udp6 socket (since 5.2);
-        
-        **sysctl** sysctl access (since 5.2);
-        
-        **getsockopt** call to getsockopt (since 5.3);
-        
-        **setsockopt** call to setsockopt (since 5.3);
-        
-        **getpeername4** call to getpeername(2) for an inet4 socket (since 5.8);
-        
-        **getpeername6** call to getpeername(2) for an inet6 socket (since 5.8);
-        
-        **getsockname4** call to getsockname(2) for an inet4 socket (since 5.8);
-        
-        **getsockname6** call to getsockname(2) for an inet6 socket (since 5.8).
-        
-        **sock_release** closing an userspace inet socket (since 5.9).
-
-    **bpftool cgroup detach** *CGROUP* *ATTACH_TYPE* *PROG*
+
+        - **sysctl** sysctl access (since 5.2);
+
+        - **getsockopt** call to getsockopt (since 5.3);
+
+        - **setsockopt** call to setsockopt (since 5.3);
+
+        - **getpeername4** call to getpeername(2) for an inet4 socket (since 5.8);
+
+        - **getpeername6** call to getpeername(2) for an inet6 socket (since 5.8);
+
+        - **getsockname4** call to getsockname(2) for an inet4 socket (since 5.8);
+
+        - **getsockname6** call to getsockname(2) for an inet6 socket (since 5.8).
+
+        - **sock_release** closing an userspace inet socket (since 5.9).
```

Comment:
We've got two entries ending with a period. Let's remove punctuation (both `;` and `.`) at the end of the entries altogether to avoid this kind of typos in the future.

Reply:
All punctuation at the end of line was removed for itemized entries.